generated/nifake/unit_tests/test_session.py

```diff
@@ -658,14 +658,98 @@ def test_repeated_capability_method_on_specific_channel(self):
         self.patched_library.niFake_ReadFromChannel.side_effect = self.side_effects_helper.niFake_ReadFromChannel
         self.side_effects_helper['ReadFromChannel']['reading'] = test_reading
         with nifake.Session('dev1') as session:
-            value = session['3'].read_from_channel(test_maximum_time)
+            value = session.channels['3'].read_from_channel(test_maximum_time)
         self.patched_library.niFake_ReadFromChannel.assert_called_once_with(matchers.ViSessionMatcher(SESSION_NUM_FOR_TEST), matchers.ViStringMatcher('3'), matchers.ViInt32Matcher(test_maximum_time_us), matchers.ViReal64PointerMatcher())
         assert value == test_reading
 
+    def test_repeated_capability_method_on_specific_channel_list_int(self):
```

Comment:
I think that if we test the (not yet created) function that creates the repeated capability string very thoroughly, then the tests from Session can be a lot simpler, just verify that that one function is called correctly. Would make things easier to follow: you wouldn't need so many verbose unit tests here.